twake/frontend/src/app/components/search-popup/search-popup.js

```diff
@@ -143,119 +141,75 @@ export default class SearchPopup extends React.Component {
-            </div>
-            <div className="header-close-btn" onClick={this.onCloseButtonClick}>
-              <img className="dismiss-btn"
-                   src="data:image/svg+xml;base64, PHN2ZyB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggb3BhY2l0eT0iMC4xMiIgZD0iTTEyIDI0QzE4LjYyNzQgMjQgMjQgMTguNjI3NCAyNCAxMkMyNCA1LjM3MjU4IDE4LjYyNzQgMCAxMiAwQzUuMzcyNTggMCAwIDUuMzcyNTggMCAxMkMwIDE4LjYyNzQgNS4zNzI1OCAyNCAxMiAyNFoiIGZpbGw9IiM4MThDOTkiLz4KPHBhdGggZD0iTTE2LjczNjQgNy4yNjM2QzE3LjA4NzkgNy42MTUwOCAxNy4wODc5IDguMTg0OTIgMTYuNzM2NCA4LjUzNjRMMTMuMjczIDEyTDE2LjczNjQgMTUuNDYzNkMxNy4wNTg2IDE1Ljc4NTggMTcuMDg1NCAxNi4yOTE1IDE2LjgxNyAxNi42NDQyTDE2LjczNjQgMTYuNzM2NEMxNi4zODQ5IDE3LjA4NzkgMTUuODE1MSAxNy4wODc5IDE1LjQ2MzYgMTYuNzM2NEwxMiAxMy4yNzNMOC41MzY0IDE2LjczNjRDOC4xODQ5MiAxNy4wODc5IDcuNjE1MDggMTcuMDg3OSA3LjI2MzYgMTYuNzM2NEM2LjkxMjEzIDE2LjM4NDkgNi45MTIxMyAxNS44MTUxIDcuMjYzNiAxNS40NjM2TDEwLjcyNyAxMkw3LjI2MzYgOC41MzY0QzYuOTQxNDIgOC4yMTQyMSA2LjkxNDU3IDcuNzA4NTMgNy4xODMwNiA3LjM1NTc3TDcuMjYzNiA3LjI2MzZDNy42MTUwOCA2LjkxMjEzIDguMTg0OTIgNi45MTIxMyA4LjUzNjQgNy4yNjM2TDEyIDEwLjcyN0wxNS40NjM2IDcuMjYzNkMxNS44MTUxIDYuOTEyMTMgMTYuMzg0OSA2LjkxMjEzIDE2LjczNjQgNy4yNjM2WiIgZmlsbD0iIzgxOEM5OSIvPgo8L3N2Zz4K"/>
+      <div className="search-popup">
+        <div className="overlay" onClick={this.onCloseButtonClick}></div>
+
+        <div className="component-container">
+          <div className="search screen">
+            <div className="header-wrapper">
+              <div className="header-title-wrapper">
+                <div className="header-title">Search</div>
+              </div>
+              <div className="header-close-btn" onClick={this.onCloseButtonClick}>
+                <img
+                  className="dismiss-btn"
+                  src="data:image/svg+xml;base64, PHN2ZyB3aWR0aD0iMjQiIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggb3BhY2l0eT0iMC4xMiIgZD0iTTEyIDI0QzE4LjYyNzQgMjQgMjQgMTguNjI3NCAyNCAxMkMyNCA1LjM3MjU4IDE4LjYyNzQgMCAxMiAwQzUuMzcyNTggMCAwIDUuMzcyNTggMCAxMkMwIDE4LjYyNzQgNS4zNzI1OCAyNCAxMiAyNFoiIGZpbGw9IiM4MThDOTkiLz4KPHBhdGggZD0iTTE2LjczNjQgNy4yNjM2QzE3LjA4NzkgNy42MTUwOCAxNy4wODc5IDguMTg0OTIgMTYuNzM2NCA4LjUzNjRMMTMuMjczIDEyTDE2LjczNjQgMTUuNDYzNkMxNy4wNTg2IDE1Ljc4NTggMTcuMDg1NCAxNi4yOTE1IDE2LjgxNyAxNi42NDQyTDE2LjczNjQgMTYuNzM2NEMxNi4zODQ5IDE3LjA4NzkgMTUuODE1MSAxNy4wODc5IDE1LjQ2MzYgMTYuNzM2NEwxMiAxMy4yNzNMOC41MzY0IDE2LjczNjRDOC4xODQ5MiAxNy4wODc5IDcuNjE1MDggMTcuMDg3OSA3LjI2MzYgMTYuNzM2NEM2LjkxMjEzIDE2LjM4NDkgNi45MTIxMyAxNS44MTUxIDcuMjYzNiAxNS40NjM2TDEwLjcyNyAxMkw3LjI2MzYgOC41MzY0QzYuOTQxNDIgOC4yMTQyMSA2LjkxNDU3IDcuNzA4NTMgNy4xODMwNiA3LjM1NTc3TDcuMjYzNiA3LjI2MzZDNy42MTUwOCA2LjkxMjEzIDguMTg0OTIgNi45MTIxMyA4LjUzNjQgNy4yNjM2TDEyIDEwLjcyN0wxNS40NjM2IDcuMjYzNkMxNS44MTUxIDYuOTEyMTMgMTYuMzg0OSA2LjkxMjEzIDE2LjczNjQgNy4yNjM2WiIgZmlsbD0iIzgxOEM5OSIvPgo8L3N2Zz4K"
+                />
```

Comment:
I don't know if it is from you originally or not, but if it is could you may be remove this big base64 stuff ?